flytekit/taskplugins/sagemaker/hpo.py

```diff
@@ -62,21 +61,20 @@ def __init__(
 
         self._task_config = task_config
         self._training_task = training_task
-        iface = training_task.python_interface
```

Comment:
why the move?  did mypy pick up on this?

Reply:
no, cleaner to have it next to where we use it